package.json

```diff
@@ -11,55 +11,55 @@
     "@types/axios": "^0.14.0",
     "@types/csv-parse": "^1.1.11",
     "@types/expect": "^1.20.3",
-    "@types/jest": "^22.1.3",
+    "@types/jest": "^22.2.3",
     "@types/moment": "^2.13.0",
-    "@types/redis": "^2.8.6",
+    "@types/redis": "^2.8.8",
     "@types/redis-mock": "^0.17.0",
-    "@types/stellar-sdk": "^0.8.1",
-    "@types/supertest": "^2.0.4",
+    "@types/stellar-sdk": "^0.8.6",
+    "@types/supertest": "^2.0.7",
     "axios": "^0.18.0",
     "axios-retry": "^3.1.1",
-    "body-parser": "^1.18.2",
+    "body-parser": "^1.18.3",
     "cookie-parser": "^1.4.3",
-    "csv-parse": "^2.0.4",
-    "express": "^4.16.2",
+    "csv-parse": "^2.5.0",
+    "express": "^4.16.4",
     "express-async-errors": "^2.1.2",
-    "express-bearer-token": "^2.1.1",
-    "hot-shots": "^5.2.0",
-    "jest": "^22.4.2",
-    "jsonwebtoken": "^8.1.1",
-    "moment": "^2.20.1",
-    "npm": "^5.8.0",
+    "express-bearer-token": "^2.2.0",
+    "hot-shots": "^5.9.1",
+    "jest": "^22.4.4",
+    "jsonwebtoken": "^8.4.0",
+    "moment": "^2.23.0",
+    "npm": "^5.10.0",
     "pg": "7.4.3",
     "pg-hstore": "2.3.2",
     "redis": "^2.8.0",
     "redis-mock": "^0.21.0",
     "redlock": "^3.1.2",
     "reflect-metadata": "^0.1.12",
-    "source-map-support": "^0.5.4",
-    "stellar-sdk": "^0.8.0",
-    "supertest": "^3.0.0",
+    "source-map-support": "^0.5.9",
+    "stellar-sdk": "^0.8.2",
+    "supertest": "^3.3.0",
     "to": "^0.2.9",
-    "ts-jest": "^22.0.4",
-    "typeorm": "^0.1.12",
+    "ts-jest": "^22.4.6",
+    "typeorm": "^0.1.21",
     "update": "^0.7.4",
-    "uuid": "^3.2.1",
-    "uuid4": "^1.0.0",
-    "winston": "^2.4.0"
+    "uuid": "^3.3.2",
+    "uuid4": "^1.1.4",
+    "winston": "^2.4.4"
   },
   "devDependencies": {
-    "@types/body-parser": "^1.16.8",
-    "@types/express": "^4.11.0",
-    "@types/jsonwebtoken": "^7.2.5",
-    "@types/redlock": "^3.0.1",
-    "@types/sequelize": "^4.27.4",
-    "@types/winston": "^2.3.7",
-    "npm-run-all": "^4.0.1",
+    "@types/body-parser": "^1.17.0",
+    "@types/express": "^4.16.0",
+    "@types/jsonwebtoken": "^7.2.8",
+    "@types/redlock": "^3.0.2",
+    "@types/sequelize": "^4.27.33",
+    "@types/winston": "^2.4.4",
+    "npm-run-all": "^4.1.5",
```

Comment:
why did we update versions?

Reply:
Because some of the modules are out of date.